src/challenge/views.py

```diff
@@ -289,6 +291,17 @@ def post(self, request):
             if challenge.first_blood is None:
                 challenge.first_blood = user
                 challenge.save()
+                hook = os.getenv("BLOOD_HOOK")
+                if hook:
+                    chall_clean = challenge.name.replace('`', '')
+                    team_clean = team.name.replace('`', '')
+                    body = {
```

Comment:
I feel like this would be better off as a Slack webhook so its not locked to one platform, Discord is compatible via adding /slack on the end of the webhook url. This does remove our ability to use `allowed_mentions` to sanitize the message but I think the same can be achieved with just replacing all `@` with `@[zero width space]`. Also, this might look better as an embed.

Reply:
I went with using Slack by default, and a Discord-style webhook if the url is a Discord one. The `/slack` endpoint didn't seem to support embeds, I could be wrong.